tests/libcxx/expected_results.txt

```diff
@@ -422,7 +392,7 @@ std/utilities/smartptr/unique.ptr/unique.ptr.class/unique.ptr.asgn/move_convert.
 std/utilities/smartptr/unique.ptr/unique.ptr.class/unique.ptr.asgn/move.pass.cpp FAIL
 
 # Test bug after LWG-3257 "Missing feature testing macro update from P0858" was accepted.
-std/language.support/support.limits/support.limits.general/string.version.pass.cpp FAIL
+std/language.support/support.limits/support.limits.general/string.version.compile.pass.cpp FAIL

```

Comment:
Test error now says "__cpp_lib_ranges_to_container should be defined in c++2b".

Reply:
This should move to the "features we don't implement yet" section and cite P1206R7 `ranges::to`.